app/components/forms/wizard/other-details-step.js

```diff
@@ -0,0 +1,160 @@
+import Component from '@ember/component';
+import { later } from '@ember/runloop';
+import { observer, computed } from '@ember/object';
+import moment from 'moment';
+import { merge, orderBy, filter, find } from 'lodash-es';
+import { licenses } from 'open-event-frontend/utils/dictionary/licenses';
+import { timezones } from 'open-event-frontend/utils/dictionary/date-time';
+import { paymentCountries, paymentCurrencies } from 'open-event-frontend/utils/dictionary/payment';
+import { countries } from 'open-event-frontend/utils/dictionary/demography';
+import FormMixin from 'open-event-frontend/mixins/form';
+import { inject as service } from '@ember/service';
+import EventWizardMixin from 'open-event-frontend/mixins/event-wizard';
+import { protocolLessValidUrlPattern } from 'open-event-frontend/utils/validators';
+import ENV from 'open-event-frontend/config/environment';
```

Comment:
Codacy found an issue: ['ENV' is defined but never used.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=6089057)